game/source_engine_nav.ksy

```diff
@@ -0,0 +1,468 @@
+# Base NAV file format in kaitai struct.
+meta:
+  id: source_engine_nav
+  title: NAV
+  file-extension: nav
+#  tags:
+#    - nav_mesh
+#    - source_engine
+#    - valve
+  license: CC0-1.0
+  endian: le
+
+doc: |
+  Base file format used by the Source Engine to store nav mesh data for maps. This KSY file does *not* account for derivative implementations.
```

Comment:
Why are you so sure? Non-literal elements of software are also copyrightable.

Reply:
@KOLANICH:

> Non-literal elements of software are also copyrightable.

Nobody talked about _copyrightability_ here, please read the initial comment (https://github.com/kaitai-io/kaitai_struct_formats/pull/561#issue-1051681769):

> (...); it does _not_ handle derivative implementations used by almost all Source Games.

Reply:
I mean that the source SDK is under a non-free license https://github.com/ValveSoftware/source-sdk-2013/blob/master/LICENSE requiring the software derived from the SDK being non-free. So if this spec is derived from the SDK source or object code, it is required to be under a non-free license.

I have no formal legal education and nothing said by me can constitute an attorney-client relationship.

Reply:
What I meant by that statement is that the KSY file does not account for differences in the NAV format made by programs that have their custom implementation of the base NAV format, such as the Counter-Strike video game series, Team Fortress 2, Left 4 Dead, and Left 4 Dead 2.

I have never implied any legal intent or meaning in that statement.